src/modules/ethereum/tradeTransactions.js

```diff
@@ -42,12 +42,12 @@ export async function fetchTradesTx(account, fromBlock, toBlock) {
         const logs = await Promise.all([
             ...logsNewOrder.map(eventLog => _formatTradeLog(NewOrder, account, eventLog)),
             ...logsCanceledOrder.map(eventLog => _formatTradeLog(CancelledOrder, account, eventLog)),
-            ...logsOrderFillBuy.map(eventLog => _formatTradeLog(OrderFill, account, eventLog, 'buy')),
-            ...logsOrderFillSell.map(eventLog => _formatTradeLog(OrderFill, account, eventLog, 'sell'))
+            ...logsOrderFillBuy.map(eventLog => _formatTradeLog(OrderFill, account, eventLog, "buy")),
+            ...logsOrderFillSell.map(eventLog => _formatTradeLog(OrderFill, account, eventLog, "sell"))
         ]);
 
         logs.sort((log1, log2) => {
```

Comment:
arrow function returns automatically so both the curly parentheses and the return keyword are redundant
```
logs.sort((log1, log2) => log2.blockData.timestamp - log1.blockData.timestamp);
```